batchUpload.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+subdomain=''
+email=''
+password=''
```

Comment:
I can foresee someone editing this file and accidentally committing it exposing their details.

Perhaps call this file something like batchUploadExample.sh, ignore batchUpload.sh from Git, and instruct people to copy the former to the latter and edit that.